src/services/contentProgress.js

```diff
@@ -274,43 +316,78 @@ async function trackProgress(contentId, collection, currentSeconds, mediaLengthS
 }
 
 export async function contentStatusCompleted(contentId, collection = null) {
-  return setStartedOrCompletedStatus(normalizeContentId(contentId), normalizeCollection(collection), true)
+  return setStartedOrCompletedStatus(
+    normalizeContentId(contentId),
+    normalizeCollection(collection),
+    true
+  )
 }
 
 export async function contentsStatusCompleted(contentIds, collection = null) {
-  return setStartedOrCompletedStatuses(normalizeContentIds(contentIds), normalizeCollection(collection), true)
+  return setStartedOrCompletedStatuses(
+    normalizeContentIds(contentIds),
+    normalizeCollection(collection),
+    true
+  )
 }
 
 export async function contentStatusStarted(contentId, collection = null) {
-  return setStartedOrCompletedStatus(normalizeContentId(contentId), normalizeCollection(collection), false)
+  return setStartedOrCompletedStatus(
+    normalizeContentId(contentId),
+    normalizeCollection(collection),
+    false
+  )
 }
 export async function contentStatusReset(contentId, collection = null) {
   return resetStatus(contentId, collection)
 }
 
 async function saveContentProgress(contentId, collection, progress, currentSeconds) {
-  const response = await db.contentProgress.recordProgress(contentId, collection, progress, currentSeconds)
+  const response = await db.contentProgress.recordProgress(
```

Comment:
Lots of formatting changes.
Main changes are here in saveContentProgress and setStartedOrCompletedStatus